pkg/sql/logictest/testdata/logic_test/pg_catalog

```diff
@@ -992,6 +1002,35 @@ JOIN pg_constraint ON objid=pg_constraint.oid AND refobjid=pg_constraint.conindi
 contype
 f
 
+# Testing table-view dependencies in pg_depend, This query will not work the same way
+# In PostgreSQL as pg_depend.classid points to pg_rewrite then in pg_rewrite
+# ev_class have dependent object, but cockroach db does not implements
+# pg_rewrite yet.
+
+statement ok
+CREATE TABLE source_table(a INT, b INT, c INT);
+CREATE VIEW depend_view AS SELECT b, c FROM source_table;
+
+query TTTTT
```

Comment:
can you make this `query TTTTT colnames` so that the output includes the column names?

Reply:
Done